src/recensio/plone/browser/review.py

```diff
@@ -229,15 +119,17 @@ def get_doi_url_if_active(self):
     def get_metadata(self):  # noqa: C901
         context = self.context
         meta = {}
-        fields = []  # XXX
-        for field in self.metadata_fields:
+
+        for field in self.context.metadata_fields:
```

Comment:
Please keep the `metadata_fields` in the view. That attribute is not used in any other place and has to stay in the view

Reply:
i wonder if we need this at all.
but for now i can move these definitions into the view.